### PR TITLE
Corrected script for some item #24

### PR DESCRIPTION
**Pull request by idk-whoami**
* **Addressed Issue(s)**: 

* **Server Mode**: re

* **Description of Pull Request**: Implementations + corrections of a bunch of items

### CODE REVIEW

File: db/re/item_db_equip.yml [hunk 1]
```diff
@@ -159442,6 +159743,178 @@ Body:
...
+      NoMail: true
+      NoAuction: true
+    ArmorLevel: 1
+    EquipLevelMin: 50
+    Refineable: true
+    View: 1070
+    Script: |
+      bonus bAgi,2;
+      bonus bAspdRate,10;
+      autobonus "{ bonus bCritical,100; bonus bLongAtkRate,5+max(0,getrefine()-6); }",50+(.@r*2),5000,BF_NORMAL,"{ active_transform 1150,5000; }";
```
Comment (Atemo) [inline db/re/item_db_equip.yml:159867]:
```suggestion
      autobonus "{ .@r = getrefine(); bonus bCritical,100; bonus bLongAtkRate,5+max(0,.@r-6); }",50+(.@r*2),5000,BF_NORMAL,"{ active_transform 1150,5000; }";
```

File: db/re/item_db_equip.yml [hunk 2]
```diff
@@ -159728,6 +160400,92 @@ Body:
...
+    Script: |
+      bonus bCrt,3+getrefine();
+      bonus2 bSubRace,RC_Player_Doram,13;
+      bonus2 bSubRace,RC_Player_Human,13;
+      bonus2 bSubClass,Class_Boss,20;
+      bonus2 bSubClass,Class_Normal,20;
+      bonus bAtkRate,40;
+      bonus2 bSkillAtk,"DK_SERVANT_W_PHANTOM",BaseLevel/10;
+      bonus2 bSkillAtk,"DK_SERVANT_W_DEMOL",BaseLevel/10;
+      bonus2 bSkillAtk,"DK_SERVANTWEAPON",BaseLevel/10;
```
Comment (Atemo) [inline db/re/item_db_equip.yml:160456]: With `bSkillAtk` the skill should always be `DK_SERVANTWEAPON_ATK` according to the data extracted from `EquipmentProperties.lub`. `DK_SERVANTWEAPON` is still used with other bonuses.
```suggestion
      bonus2 bSkillAtk,"DK_SERVANTWEAPON_ATK",BaseLevel/10;
```

File: db/re/item_db_equip.yml [hunk 3]
```diff
@@ -181015,14 +182133,98 @@ Body:
...
+      Garment: true
+    ArmorLevel: 1
+    EquipLevelMin: 50
+    Refineable: true
+    Script: |
+      .@r = getrefine();
+      bonus bMaxHP,100;
+      bonus bMaxSP,10;
+      bonus bLongAtkDef,5*.@r;
+      if (BaseJob == Job_Creator || BaseJob == Job_Alchemist)
```
Comment (Atemo) [inline db/re/item_db_equip.yml:182175]:
```suggestion
      if (BaseJob == Job_Alchemist)
```

File: db/re/item_db_equip.yml [hunk 4]
```diff
@@ -181015,14 +182133,98 @@ Body:
...
+      NoAuction: true
+    ArmorLevel: 1
+    EquipLevelMin: 50
+    Refineable: true
+    Script: |
+      .@r = getrefine();
+      bonus bMaxHP,100;
+      bonus bMaxSP,10;
+      bonus bLongAtkDef,5*.@r;
+      if (BaseJob == Job_Creator || BaseJob == Job_Alchemist)
```
Comment (Atemo) [inline db/re/item_db_equip.yml:182202]:
```suggestion
      if (BaseJob == Job_Alchemist)
```

File: db/re/item_db_equip.yml [hunk 5]
```diff
@@ -185945,6 +187322,504 @@ Body:
...
+      Both_Accessory: true
+    ArmorLevel: 1
+    EquipLevelMin: 55
+    Script: |
+      bonus bStr,3;
+      bonus bAgi,3;
+      bonus bMaxSP,75;
+      bonus bMdef,15;
+      bonus3 bAutoSpell,"MO_CALLSPIRITS",5,1000;
+      if (BaseJob == Job_Monk || BaseJob == Job_Champion) {
```
Comment (Atemo) [inline db/re/item_db_equip.yml:187410]:
```suggestion
      if (BaseJob == Job_Monk) {
```

File: db/re/item_db_equip.yml [hunk 6]
```diff
@@ -159728,6 +160400,92 @@ Body:
...
+      NoAuction: true
+    ArmorLevel: 1
+    EquipLevelMin: 55
+    Script: |
+      bonus bStr,3;
+      bonus bAgi,3;
+      bonus bMaxSP,75;
+      bonus bMdef,15;
+      bonus3 bAutoSpell,"MO_CALLSPIRITS",5,1000;
+      if (BaseJob == Job_Monk || BaseJob == Job_Champion) {
```
Comment (Atemo) [inline db/re/item_db_equip.yml:160481]: BaseJob always returns the non transcendent job
```suggestion
      if (BaseJob == Job_Monk) {
```

File: db/re/item_combos.yml [hunk 7]
```diff
@@ -43416,3 +43418,1153 @@ Body:
...
+      }
+  - Combos:
+      - Combo:
+          - Remorse_C_Crown    # 400440
+          - Poenitentia_Termes    # 640014
+    Script: |
+      .@r = getequiprefinerycnt(EQI_HAND_R);
+      bonus bSMatk,10;
+      bonus2 bSkillAtk,"AG_RAIN_OF_CRYSTAL",10*(.@r/2);
+      bonus2 bSkillAtk,"AG_CRIMSON_ARROW",10*(.@r/2);
```
Comment (Atemo) [inline db/re/item_combos.yml:44218]: According to the data extracted from `EquipmentProperties` `AG_CRIMSON_ARROW` and `AG_CRIMSON_ARROW_ATK` are always together for `bSkillAtk`
```suggestion
      bonus2 bSkillAtk,"AG_CRIMSON_ARROW",10*(.@r/2);
      bonus2 bSkillAtk,"AG_CRIMSON_ARROW_ATK",10*(.@r/2);
```

**Review by Atemo — COMMENTED** [review]
Look good!

File: db/re/item_db_equip.yml (hunk 6 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml:160481]: Thanks !

File: db/re/item_db_equip.yml (hunk 5 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml:187410]: Thanks !

File: db/re/item_db_equip.yml (hunk 4 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml:182202]: Thanks !

File: db/re/item_db_equip.yml (hunk 3 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml:182175]: Thanks !

File: db/re/item_db_equip.yml (hunk 1 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml:159867]: Thanks !

File: db/re/item_db_usable.yml [hunk 8]
```diff
@@ -2162,7 +2162,7 @@ Body:
   - Id: 9514
     AegisName: Ein_Ddbox
     Name: Physical Modification Permit
-    Type: DelayConsume
+    Type: Delayconsume
```
[the remark this answers is not in the document]
Comment (idk-whoami) [reply db/re/item_db_usable.yml]: Oh okay i chose DelayConsume like you :)

File: db/re/item_combos.yml (hunk 7 above)
Comment (idk-whoami) [reply db/re/item_combos.yml:44218]: Oh okay, Thanks !

File: db/re/item_db_equip.yml (hunk 2 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml:160456]: Oh okay, Thanks !

**Pull request merged**